linkage/pvs_like_case_study/pvs_like_case_study_sample_data.ipynb

```diff
@@ -15,26 +15,161 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # TODO: Have this function output more charts and diagnostics
</pre><p>Since this function is so central to your following code, it would be helpful to walk the reader through it more gently</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
I've done my best in the next version, though there is still probably room for improvement in refactoring this.